neurobooth_os/iout/iphone.py

```diff
@@ -968,11 +968,12 @@ def script_capture_data(subject_id: str, recording_folder: str, capture_duration
     dev_args = IPhoneDeviceArgs(
         ENV_devices={'IPhone_dev_1': {}},
         device_id='IPhone_dev_1',
-        device_name='IPhone',
-        wearable_bool=False,
         sensor_ids=['IPhone_sens_1'],
-        sensor_array=[],  # The sensor array and arg parser are not needed by the test script
+        # sensor array, arg parser, and device start function are not needed by the
```

Comment:
If the function script_capture_data is used just for testing, could you add a comment to that effect? Otherwise, looks good.